AdminUi/src/AdminUi.Infrastructure/Persistence/Database/EntityTypeConfigurations/ClientOverviewEntityTypeConfiguration.cs

```diff
@@ -8,6 +8,13 @@ public class ClientOverviewEntityTypeConfiguration : IEntityTypeConfiguration<Cl
     public void Configure(EntityTypeBuilder<ClientOverview> builder)
     {
         builder.ToView("ClientOverviews");
-        builder.HasNoKey();
+        builder.OwnsOne(
+            c => c.DefaultTier,
+            dt =>
+            {
+                dt.Property(p => p.Id).HasColumnName("DefaultTierId");
+                dt.Property(p => p.Name).HasColumnName("DefaultTierName");
+            });
+        builder.HasKey(c => c.ClientId);
```

Comment:
I don't know whether it is a good idea to define a key here.
1. Afaik a view can't have a PK.
2. The ClientId isn't part of the PK in the Clients table. Maybe this will cause problems?

Did you run your code on SQL Server as well as Postgres?

Reply:
I ran this both on SQL Server and Postgres and everything seemed to be working properly, I didn't find any issues.

Regarding the first two questions, when I first implemented this `.OwnsOne()`, I had a `.HasNoKey()` but was running into this error when creating migrations:
![MicrosoftTeams-image](https://github.com/nmshd/backbone/assets/118175875/b5069b43-8172-4b38-8c32-9203f030e50b)
After troubleshooting with Daniel Almeida, we tried adding the clientId as key and it worked.

Looking at the migration code, at first glance, I don't think there should be any problems in the future, but I do agree that it's weird having to assign a key to a view.